spire/templates/apps/dovetail-router.yml

```diff
@@ -940,15 +953,18 @@ Resources:
         };
 
         exports.handler = async (event) => {
+          console.log(JSON.stringify(event));
```

Comment:
Temporary for the "real world test run"?

Reply:
I like logging events in Lambdas so that if I ever need to run them manually there are good samples around. I can remove though, if you want.